WalletWasabi.Gui/Global.cs

```diff
@@ -57,7 +58,8 @@ public class Global
 		public NodesGroup Nodes { get; private set; }
 		public WasabiSynchronizer Synchronizer { get; private set; }
 		public FeeProviders FeeProviders { get; private set; }
-		public WalletService WalletService { get; private set; }
+		public WalletManager WalletManager { get; }
```

Comment:
WalletManager isnt accessed outside of Global, private?

imo we should be moving away from Global, and just make WalletManager a singleton, we need to move away from passing around this "Bag" of stuff, its the "non-global global pattern".

No reason for WalletManager to be mixed up with all this other logic that ideally can be seperated into relevant interfaces and classes.

However this can be a future goal.

Reply:
It will be accessed from outside when we will start removing `WalletService` from global. 

About the other problem. It sounds reasonable but out of scope for this PR.